Dockerfile

```diff
@@ -1,222 +1,8 @@
-    fi && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-# copy Pyperf, licenses and notice file.
-COPY --from=bcc-build /bcc/root/share/bcc/examples/cpp/PyPerf gprofiler/resources/python/pyperf/
-COPY --from=bcc-build /bcc/bcc/LICENSE.txt gprofiler/resources/python/pyperf/
-COPY --from=bcc-build /bcc/bcc/licenses gprofiler/resources/python/pyperf/licenses
-COPY --from=bcc-build /bcc/bcc/NOTICE gprofiler/resources/python/pyperf/
-COPY --from=bcc-build /bpf_get_fs_offset/get_fs_offset gprofiler/resources/python/pyperf/
-COPY --from=bcc-build /bpf_get_stack_offset/get_stack_offset gprofiler/resources/python/pyperf/
-
-COPY --from=pyspy-builder /tmp/py-spy/py-spy gprofiler/resources/python/py-spy
-
-COPY --from=perf-builder /perf gprofiler/resources/perf
-
-COPY --from=phpspy-builder /tmp/phpspy/phpspy gprofiler/resources/php/phpspy
-COPY --from=phpspy-builder /tmp/binutils/binutils-2.25/bin/bin/objdump gprofiler/resources/php/objdump
-COPY --from=phpspy-builder /tmp/binutils/binutils-2.25/bin/bin/strings gprofiler/resources/php/strings
-
-COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/bin/asprof gprofiler/resources/java/asprof
-COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/async-profiler-version gprofiler/resources/java/async-profiler-version
-COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/lib/libasyncProfiler.so gprofiler/resources/java/glibc/libasyncProfiler.so
-COPY --from=async-profiler-builder-musl /tmp/async-profiler/build/lib/libasyncProfiler.so gprofiler/resources/java/musl/libasyncProfiler.so
-COPY --from=node-package-builder-musl /tmp/module_build gprofiler/resources/node/module/musl
-COPY --from=node-package-builder-glibc /tmp/module_build gprofiler/resources/node/module/glibc
-
-COPY --from=rbspy-builder /tmp/rbspy/rbspy gprofiler/resources/ruby/rbspy
-
-ENV DOTNET_ROOT=/app/gprofiler/resources/dotnet
-COPY --from=dotnet-builder /usr/share/dotnet/host gprofiler/resources/dotnet/host
-COPY --from=dotnet-builder /tmp/dotnet/deps gprofiler/resources/dotnet/shared/Microsoft.NETCore.App/6.0.7
-COPY --from=dotnet-builder /tmp/dotnet/tools gprofiler/resources/dotnet/tools
-
-COPY --from=burn-builder /tmp/burn/burn gprofiler/resources/burn
-
-# we want the latest pip
-# hadolint ignore=DL3013
-RUN pip3 install --upgrade --no-cache-dir pip
-
-# done separately from the 'pip3 install -e' below; so we don't reinstall all dependencies on each
-# code change.
-COPY requirements.txt ./
-COPY granulate-utils/setup.py granulate-utils/requirements.txt granulate-utils/README.md granulate-utils/
-COPY granulate-utils/granulate_utils granulate-utils/granulate_utils
-COPY granulate-utils/glogger granulate-utils/glogger
-RUN pip3 install --no-cache-dir -r requirements.txt
-
-COPY LICENSE.md MANIFEST.in README.md setup.py  ./
-COPY gprofiler gprofiler
-RUN pip3 install --no-cache-dir -e .
-
+FROM alpine as gprofiler
```

Comment:
Some spacing would make in this file to make it readable. between the `FROM` to next lines, before `ENTRYPOINT`.